Apps/WebClient/src/NewClientApp/src/components/public/landing/LandingView.vue

```diff
@@ -55,17 +50,37 @@ const oidcIsAuthenticated = computed<boolean>(
 const offlineMessage = computed(
     () => configStore.webConfig.offlineMode?.message ?? ""
 );
-const proofOfVaccinationTile = computed<Tile>(() => ({
+const proofOfVaccinationTile = computed<InfoTile>(() => ({
     type: "ProofOfVaccination",
     icon: "check-circle",
+    logoUrl: new URL("@/assets/images/gov/canada-gov-logo.svg", import.meta.url)
+        .href,
     name: "Proof of Vaccination",
-    description: "View and download your proof of vaccination",
+    description: "View and download your proof of vaccination.",
     active: configStore.webConfig.featureToggleConfiguration.covid19
         .publicCovid19.showFederalProofOfVaccination,
 }));
-const tiles = computed(() => {
-    // Get core tiles from entry type constants
-    const tiles = entryTypes.map<Tile>((type) => {
+const organDonorRegistrationTile = computed<InfoTile>(() => ({
+    type: "OrganDonorRegistration",
+    icon: "check-circle",
+    logoUrl: new URL("@/assets/images/services/odr-logo.svg", import.meta.url)
+        .href,
+    name: "Organ Donor Registration",
+    description:
+        "Check your Organ Donor Registration status with BC Transplant.",
+    active: ConfigUtil.isServiceEnabled(ServiceName.OrganDonorRegistration),
+}));
+const servicesTiles = computed(() => {
+    return [
+        proofOfVaccinationTile.value,
+        organDonorRegistrationTile.value,
+    ].filter((tile) => tile.active);
+});
```

Comment:
Arrow function could be converted to use expression instead of braces and `return` statement.